### PR TITLE
Add changeAdminStatus

### PR DESCRIPTION
**Pull request by AstroCB**
Adds a `changeAdminStatus` function to allow the API to alter the admin status of users in group chats (naming suggestions welcome).

re: #624

### CODE REVIEW

**Comment by ravkr** [comment]
could you do
```bash
git remote add upstream https://github.com/Schmavery/facebook-chat-api.git
git rebase upstream/master
git push -f
```
it would get rid of all these duplicated commits in every pull request :/

**Comment by AstroCB** [comment]
Good call – thanks @ravkr

File: src/changeAdminStatus.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+module.exports = function(defaultFuncs, api, ctx) {
+  return function changeAdminStatus(userIDs, makeAdmin, threadID, callback) {
+    if (
+      !callback &&
+      (utils.getType(makeAdmin) === "AsyncFunction" ||
+        utils.getType(makeAdmin) === "AsyncFunction" ||
+        utils.getType(threadID) === "Function" ||
+        utils.getType(threadID) === "AsyncFunction")
+    ) {
+      throw { error: "please pass makeAdmin and threadID as the second/third arguments." };
```
Comment (Schmavery) [inline src/changeAdminStatus.js]: If these are required fields, we can probably just check their types rather than checking if they're not a callback?

**Comment by Schmavery** [comment]
Thanks @AstroCB 
@ravkr does this PR do the same thing as #659? What's your concern with this one? I don't see much difference other than extra fields in the form, can someone confirm whether these are sent by the browser? If so, we should mimic that.
Or is there something else I'm missing?

**Comment by ravkr** [comment]
Yes, both PR are doing the same thing
But mine looks a bit cleaner (no extra fields) 
I copied that request from messenger.com

**Comment by Schmavery** [comment]
@AstroCB did you find these fields are required/used by the browser when you looked or do you think we can just remove them?

**Comment by AstroCB** [comment]
Nope, they’re just holdovers from another file that I used as the basis for this – should be good to remove.

File: src/changeAdminStatus.js (hunk 1 above)
Comment (AstroCB) [reply src/changeAdminStatus.js]: Yep, another holdover from the basis file I used.

**Comment by Schmavery** [comment]
@ravkr 's change has no docs though 😛
At this point idk which one to spend time reviewing, if you two want to decide if/how you want to split this up, I don't see a problem with adding this functionality.

**Comment by ravkr** [comment]
You can review mine PR
I'll add docs soon

**Comment by AstroCB** [comment]
Made the discussed changes last night and forgot to publish – they're up now, but either way works for me.

**Comment by ravkr** [comment]
@Schmavery what's the decision?

**Comment by Schmavery** [comment]
Thanks for helping out @AstroCB, we ended up merging @ravkr's branch <3

**Pull request closed without merging**